styles.css

```diff
@@ -0,0 +1,141 @@
+/* General Styles */
+body {
+    font-family: 'Open Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+    background-image: url('bg.jpg'); /* Replace with your image URL */
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-attachment: fixed; /* Keeps the background fixed while scrolling */
+    color: #333;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+/* Header Styles */
+header {
+    background-color: rgba(1, 5, 28, 0.806); /* Semi-transparent background to blend with the image */
+    color: #f5f5f5;
+    padding: 20px 40px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+header h1 {
+    font-family: 'Playfair Display', serif;
+    font-size: 2rem;
+    margin: 0;
+}
+
+nav a {
+    color: #f5f5f5;
+    margin: 0 15px;
+    text-decoration: none;
+    font-weight: 600;
+    transition: color 0.3s ease;
+}
+
+nav a:hover {
+    color: #e0a80d;
+}
+
+/* Hero Section Styles */
+.hero {
+    background-color: #abdff48c; /* Solid background color */
+    color: #080000;
+    padding: 60px 20px;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.5);
+}
+
+
+.hero h2 {
+    font-family: 'Playfair Display', serif;
+    font-size: 2.5rem;
+    margin: 0;
+    max-width: 800px;
+    line-height: 1.4;
+}
+
+/* Form Section Styles */
+.form-section {
+    padding: 40px;
+    max-width: 500px;
+    margin: 30px auto;
+    background-color: rgba(165, 222, 250, 0.381); /* Slightly transparent for a subtle effect */
+    box-shadow: 0 4px 10px rgba(1, 18, 46, 0.1);
+    border-radius: 8px;
+}
+
+.form-section form {
+    display: flex;
+    flex-direction: column;
+}
+
+.form-section label {
+    margin-bottom: 8px;
+    font-weight: 600;
+    color: #434243;
+}
+
+.form-section input {
+    padding: 12px;
+    margin-bottom: 20px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    font-size: 1rem;
+}
+
+.form-section button {
+    padding: 14px;
+    border: none;
+    background-color: #050120;
+    color: #ffffff;
+    font-size: 1.2rem;
+    cursor: pointer;
+    border-radius: 4px;
+    transition: background-color 0.3s ease, transform 0.2s ease;
+}
+
+.form-section button:hover {
+    background-color: #333;
+    transform: scale(1.05);
+}
+
+/* Footer Styles */
+footer {
+    background-color: rgba(1, 5, 28, 0.806); /* Semi-transparent background to blend with the image */
+    color: #f5f5f5;
+    text-align: center;
+    padding: 20px 40px;
+    margin-top: auto;
+}
+
+/* Responsive Design */
+@media (max-width: 768px) {
```

Comment:
Nice!!
Did you test this for different screens? If yes, great!! If not, feel free to check how it looks on different screens such as mobile, ipad, desktop etc. using developer tools.